src/middlewares/Fetch.ts

```diff
@@ -15,8 +17,11 @@ export type FetchResponse<T> = {
     statusText: string;
     type: string;
     url: string;
+    cacheUsed?: boolean;
```

Comment:
i'm strongly in favor of putting all options related to a single middleware into a single property, to keep changes to Fetch.ts to a minimum and middlewares as interoperable as possible.